arch/arm/include/spinlock.h

```diff
@@ -95,6 +95,11 @@
 #define SP_DSB(n) __asm__ __volatile__ ("dsb sy" : : : "memory")
 #define SP_DMB(n) __asm__ __volatile__ ("dmb st" : : : "memory")
 
+#ifdef CONFIG_ARM_WFE_SEV
```

Comment:
why not always define SP_WFE/SP_SEV instead?

Reply:
> why not always define SP_WFE/SP_SEV instead like SP_DSB/SP_DMB?

Because some processors (e.g. cxd56xx) do not support the instructions.

Reply:
> > why not always define SP_WFE/SP_SEV instead like SP_DSB/SP_DMB?
> 
> Because some processors (e.g. cxd56xx) do not support the instructions.

More specifically, cxd56xx supports the wfe instruction but does not work with the sev instruction.

Reply:
If so, why not move the define to arch/armv7-a/spinlock.h?

Reply:
> If so, why not move the define to arch/armv7-a/spinlock.h?

Because LC823450 (dual Cortex-M3) supports the instructions. (not tested yet but described in the document)

Reply:
Ok